src/components/contact-list.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Contact from './contact.js';
+
+export default class ContactList extends React.Component {
+  render() {
+    const contacts = this.props.data;
+
+    return (
+      <ul className="contact-list">
+        {contacts.map((contact,i) => {
+          return (
+            <Contact {...contact} handleSelect={this.props.handleSelect} key={i} searchValue={this.props.searchValue} />
+          );
+        })
+        }
+      </ul>
+    );
+  }
+}
+
+ContactList.propTypes = {
+  data: React.PropTypes.array,
+  name: React.PropTypes.string,
+  occupation: React.PropTypes.string,
+  handleSelect: React.PropTypes.isfunc
```

Comment:
Proptype should be PropTypes.func, as mentioned above. Name for functions are usually called `onSomeAction`.